setup.py

```diff
@@ -51,9 +51,7 @@
     },
     python_requires=">=3.7,<3.8",
     entry_points={
-        "console_scripts": [
-            "openfisca-us = openfisca_us.tools.cli:main",
-        ],
+        "console_scripts": ["openfisca-us = openfisca_us.tools.cli:main",],
```

Comment:
I think this will fix the lint
```suggestion
        "console_scripts": ["openfisca-us = openfisca_us.tools.cli:main",]
```